client/src/components/Monument/Details/About/About.js

```diff
@@ -12,12 +12,109 @@ export default class About extends React.Component {
 
         const { monument, contributions, references } = this.props;
 
-        let lastUpdated = (
-            <div>
-                <span className="detail-label">Last Updated:&nbsp;</span>
-                {this.prettyPrintDate(monument.updatedDate)}
-            </div>
-        );
+        let title;
+        if (monument.title) {
+            title = (
+                <div>
+                    <span className='detail-label'>Title:&nbsp;</span>
+                    {monument.title}
+                </div>
+            );
+        }
+
+        let artist;
+        if (monument.artist) {
+            artist = (
+                <div>
+                    <span className='detail-label'>Artist:&nbsp;</span>
+                    {monument.artist}
+                </div>
+            );
+        }
+
+        let date;
+        if (monument.date) {
+            date = (
+                <div>
+                    <span className='detail-label'>Date:&nbsp;</span>
+                    {this.prettyPrintDate(monument.date)}
+                </div>
+            );
+        }
+
+        let city;
+        if (monument.city) {
+            city = (
+                <div>
+                    <span className='detail-label'>City:&nbsp;</span>
+                    {monument.city}
+                </div>
+            );
+        }
+
+        let state;
+        if (monument.state) {
+            state = (
+                <div>
+                    <span className='detail-label'>State:&nbsp;</span>
+                    {monument.state}
+                </div>
+            );
+        }
+
+        let address;
+        if (monument.address) {
+            address = (
+                <div>
+                    <span className='detail-label'>Address:&nbsp;</span>
+                    {monument.address}
+                </div>
+            )
+        }
```

Comment:
Could do this in a loop but eh... this is probably simpler

Reply:
:shrug: